op-deployer/pkg/deployer/testutil/env.go

```diff
@@ -15,6 +16,19 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func IsolatedTestDirWithAutoCleanup(t *testing.T) string {
```

Comment:
Why not rename the env var to something more generic, then move this into `op-service`? That way you avoid the circular dependency and can remove the duplication.

Reply:
good idea, done!